Automattic/jetpack
# PHPCS: SAL (part 1)

This PR partially fixes the issue at #21967

#### Changes proposed in this Pull Request:

* This PR updates the SAL files to meet PHPCS standards:
  * "projects/plugins/jetpack/sal/class.json-api-date.php"
  * "projects/plugins/jetpack/sal/class.json-api-links.php" 
  * "projects/plugins/jetpack/sal/class.json-api-metadata.php"
  * "projects/plugins/jetpack/sal/class.json-api-platform-jetpack.php"
  * "projects/plugins/jetpack/sal/class.json-api-platform.php"


#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* For each file, test features impacted by that file. For class.json-api-date.php for example, the date of a published post when viewing the WordPress.com posts page (https://wordpress.com/posts/sitename.com) for a Jetpack connected site will be affected by changes to the date functionality here.
* Also, you can use the WordPress.com developer console to test an endpoint such as the WP.COM API GET request for `/sites/{blog_id}/posts/{post_id}` - for a sandboxed WPcom site and a sandboxed Jetpack site. A breaking change will affect the output, whereas it should work fine otherwise.
* We're primarily checking for any broken experiences or PHPCS errors.
